test/data/decodedTransactions.js

```diff
@@ -0,0 +1,100 @@
+export const decodedPurchasedTicketTx = {
+  "version": 1,
+  "serType": 0,
+  "numInputs": 1,
+  "inputs": [
+    {
+      "opRawHash": Buffer.from([132, 107, 192, 162, 131, 217, 8, 228, 140, 36, 137, 147, 128, 220, 21, 80, 233, 149, 72, 230, 4, 157, 215, 148, 83, 194, 10, 148, 245, 249, 167, 105]),
```

Comment:
I'd recommend creating a helper to convert from an hex string to a Buffer for these functions, since it makes it easier to review.

Or maybe dropping `opRawHash` altogether and working off of `prevTxId` (since having two fields for the same info could lead to using inconsistent data)

Reply:
Agree. I removed the opRawHash from decodedTransaction object.